app/models/organization.rb

```diff
@@ -1,12 +1,12 @@
 class Organization < ActiveRecord::Base
   include RoleControl::Owned
-  include RoleControl::Controlled
+  include RoleControl::PunditInterop
   include Activatable
   include Linkable
   include SluggedName
   include Translatable
 
-  scope :public_scope, -> { where(listed: true) }
+  # Still needed for HttpCacheable
```

Comment:
I assume the `HttpCacheable` can move to a different kind of scope resolver / policy to determine what should be cacheable, not in this PR but wondering if that is the plan?

Reply:
Don't really have a plan for it yet. Maybe turn it into a `http_cacheable?` method on the model? I think right now the controller is making a separate SQL call to check if there are any private resources. But it could probably do 
```ruby
def public_resources?
  resources.all? { |resource| resource.http_cacheable? }
end
```